pkg/ui/src/views/shared/components/sortabletable/index.tsx

```diff
@@ -96,11 +111,23 @@ export class SortableTable extends React.Component<TableProps, {}> {
     });
   }
 
+  expansionControl(expanded: boolean) {
+    const content = expanded ? "▼" : "▶";
```

Comment:
If these were in the stylesheet they'd be right next to the up/down arrows, which seems nice.

Reply:
Yeah, but it seems more straightforward just to put them in the code, rather than in `:after`… I dunno